cc-snapshot

```diff
@@ -119,6 +119,7 @@ if [ -f "$PROVENANCE_FILE" ]; then
   do
     prop="${prop/\":\"/=}"
     prop="$(echo $prop | tr -d '"')"
+    [[ $prop == chameleon-supported* ]] && continue
     PROVENANCE_PROPERTIES="--property provenance-$prop $PROVENANCE_PROPERTIES"
```

Comment:
although totally fine, this wasn't a bash structure I'd been familiar with.
it _miight_ be more readable if changed to e.g.
```
if [ condition ]; then
  append to properties var
fi
```
but the one liner is kinda nice.

Reply:
definitely up to you. if you think this is ok, I'm good merging but I'm more than happy to expand the if statement out too